### PR TITLE
[SDL 0305] Homogenize TextFieldName

### PR DESCRIPTION
**Pull request by iCollin**
Implements/Fixes https://github.com/smartdevicelink/sdl_hmi/issues/371

This PR is **ready** for review.

### Summary
add timeToDestination to capabilities
remove navigationText references
fix navigation template mainField assignment

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: capabilities/display_capabilities.js [hunk 1]
```diff
@@ -168,6 +168,12 @@ SDL.templateCapabilities = {
                     "width": 500,
                     "rows": 1
                 },
+                {
+                    "name": "timeToDestination",
+                    "characterSet": "UTF_8",
+                    "width": 500,
+                    "rows": 1
+                },
```
Comment (jacobkeeler) [inline capabilities/display_capabilities.js:176]: Should add turnText as well

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

File: capabilities/display_capabilities.js (hunk 1 above)
Comment (iCollin) [reply capabilities/display_capabilities.js:176]: added in https://github.com/smartdevicelink/sdl_hmi/pull/473/commits/d77ce1f760582accfb3e4f82942218ec4a704f2e

**Review by jacobkeeler — CHANGES_REQUESTED** [review]
Hmmm, it looks like there are several other places where `timeToDestination` and `turnText` will need to be added (twice in `UIRPC.js` and once in `data.js`).

That said, it's probably worth changing this so that they all get the capabilities from the same place

**Comment by iCollin** [comment]
> That said, it's probably worth changing this so that they all get the capabilities from the same place

While I agree, I looked into this and doing it properly would take a bit.
Unfortunately I feel like there is some pressure to get this feature in a good state quickly, so I have just made the necessary changes for now.

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**